x/ratelimit/keeper/rate_limit.go

```diff
@@ -288,42 +287,45 @@ func (k Keeper) GetAllBlacklistedDenoms(ctx sdk.Context) []string {
 	return allBlacklistedDenoms
 }
 
-// Adds an address to a whitelist to allow all IBC transfers to/from the address,
-// and skip all flow calculations
-func (k Keeper) AddAddressToWhitelist(ctx sdk.Context, address string) {
+// Adds an pair of sender and receiver addresses to the whitelist to allow all
+// IBC transfers between those addresses to skip all flow calculations
+func (k Keeper) SetAddressWhitelist(ctx sdk.Context, whitelist types.AddressWhitelist) {
 	store := prefix.NewStore(ctx.KVStore(k.storeKey), types.AddressWhitelistKeyPrefix)
-	key := types.KeyPrefix(address)
-	store.Set(key, []byte{1})
+	key := types.GetAddressWhitelistKey(whitelist.Sender, whitelist.Receiver)
+	value := k.cdc.MustMarshal(&whitelist)
+	store.Set(key, value)
 }
 
-// Removes an address from a whitelist so that it's transfers are counted in the quota
-func (k Keeper) RemoveAddressFromWhitelist(ctx sdk.Context, address string) {
+// Removes a whitelisted address pair so that it's transfers are counted in the quota
+func (k Keeper) RemoveAddressWhitelist(ctx sdk.Context, sender, receiver string) {
 	store := prefix.NewStore(ctx.KVStore(k.storeKey), types.AddressWhitelistKeyPrefix)
-	key := types.KeyPrefix(address)
+	key := types.GetAddressWhitelistKey(sender, receiver)
 	store.Delete(key)
 }
 
-// Check if an address is currently whitelisted
-func (k Keeper) IsAddressWhitelisted(ctx sdk.Context, address string) bool {
+// Check if an address pair is currently whitelisted
```

Comment:
Can we call them routes to make clear that they're directional? 

For example, if I read that (A,B) was a whitelisted pair, I would assume that both A->B and B->A would be excluded from rate limiting, but in reality only A->B is excluded

Reply:
Hmm that's a good point. But I think Address/Account should still be specified, otherwise `Route` may easily be conflated with a channel, and `Path` is already used to describe the channel